shell_sort/shell_sort.py

```diff
@@ -0,0 +1,37 @@
+def shell_sort(array: [int]):
+    """
+    Sorts the given array of integers using the Shell Sort algorithm
+    """
```

Comment:
Please mention the `:param` and `:return` elements in the docstring. Also, just mention the complexities in the docstring.